### PR TITLE
[Fixes #19815033 #19473719] Prevents activity managers from editing activ

### PR DESCRIPTION
**Pull request by brvtl**
[Fixes #19815033 #19473719] Prevents activity managers from editing activities/projects

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -51,6 +51,13 @@ class ApplicationController < ActionController::Base
       @current_user
     end
 
+    def check_activity_manager_permissions
+      if current_user.roles.size == 1 && current_user.activity_manager? && !current_user.sysadmin?
```
Comment (sachinr) [inline app/controllers/application_controller.rb]: As discussed - if user is am and reporter then they can edit anything

File: features/users/sysadmins/can_manage_activities.feature [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@run
```
Comment (sachinr) [inline features/users/sysadmins/can_manage_activities.feature]: remove

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (dalibor) [inline app/controllers/application_controller.rb]: @petergarbers: current_user.roles.size == 1 && current_user.activity_manager? - means that user has only 1 role and it's activity_manager. You don't need  && !current_user.sysadmin? check

File: spec/controllers/projects_controller_spec.rb [hunk 3]
```diff
@@ -158,4 +158,117 @@ describe ProjectsController do
...
+        login @user
+        session[:return_to] = edit_response_project_url(@data_response, @project)
+        put :update, :id => @project.id, :response_id => @data_response.id,
+          :project => {:description => "thedesc", :project_id => @project.id}
+
+        @project.description.should_not == "thedesc"
+      end
+    end
+
+    context "who are sysadmins and activity managers" do
```
Comment (dalibor) [inline spec/controllers/projects_controller_spec.rb:237]: use same language as other contexts: "Sysadmin and Activity Manager"

**Pull request merged**